### PR TITLE
Added Archana's bio to ruby and typescript tracks

### PR DESCRIPTION
**Pull request by archanid**

### CODE REVIEW

File: mentors/ruby/a.json [hunk 1]
```diff
@@ -39,4 +39,12 @@
     "avatar_url": null,
     "bio": "Backend developer. Love Ruby because I find it to be very intuitive, clean and easy to learn!"
   }
```
Comment (SleeplessByte) [inline mentors/ruby/a.json]:
```suggestion
  },
```

**Comment by archanid** [comment]
@SleeplessByte changed all the `link_*` to null so it picks up the default gh stuff. Same as javascript. Going to do typescript too.

**Comment by archanid** [comment]
@SleeplessByte 👀 ready for review

**Comment by SleeplessByte** [comment]
Welcome to both!

**Pull request merged**